apps-rendering/src/components/Layout/Layout.stories.tsx

```diff
@@ -61,6 +63,21 @@ export const Article = (): React.ReactNode => {
 };
 Article.story = { name: 'Article' };
 
+export const ArticleSRA = (): React.ReactNode => {
+	return (
+		<Standard item={{ ...article, theme: 7 }}>
+			{renderAll(
+				formatFromItem(
+					{ ...article, theme: 7 },
+					some(ArticleDisplay.Standard),
+				),
+				Result.partition(article.body).oks,
+			)}
+		</Standard>
+	);
+};
+ArticleSRA.story = { name: 'Article - SRA' };
```

Comment:
Just for clarity! Took me a second to expand the acronym

```suggestion
ArticleSRA.story = { name: 'Article - Special Report Alt' };
```